src/components/Search.jsx

```diff
@@ -1,19 +1,17 @@
-function Search({setArticleSearch, setArticleFilter}) {
-    
-    
-      function handleFilter() {
-        setArticleFilter(document.getElementById("topic").value);
-      }
-   
-    return ( 
-        <>
-        <select name="topic" id="topic" onChange={handleFilter}>
-          <option value="coding">Coding</option>
-          <option value="football">Football</option>
-          <option value="cooking">Cooking</option>
-        </select>
-      </>
-     );
+function Search({ setArticleFilter }) {
+  function handleFilter() {
+    setArticleFilter(document.getElementById("topic").value);
```

Comment:
We could use `event.target.value` here instead of this. You will have to pass in `event` as a parameter :)